src/app/services/articles/index.ts

```diff
@@ -46,4 +45,26 @@ const getArticlesService = (
     });
 };
 
-export default getArticlesService;
+export const getArticleByTitle = (title: string) => {
+  const url = `everything`;
+  const params = {
+    q: title,
+    pageSize: 1,
+    apiKey,
+  };
+
+  return ApiService.get(url, params)
+    .then((response: any) => {
+      const { data } = response;
+      return {
+        ...response,
+        data: {
+          status: data.status,
+          article: data.articles[0],
```

Comment:
Maybe you can use here array destructuring instead of articles[0]?
`const [item] = data.articles;`
For me, it will be much clear in code.